api/src/main/java/com/microsoft/gctoolkit/message/DataSourceBus.java

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.gctoolkit.message;
+
+public interface DataSourceBus<S> {
```

Comment:
Javadoc please, same goes for the others

Reply:
in flight...